frontend/src/constants/Routes.ts

```diff
@@ -14,6 +14,8 @@ export const UPDATE_ENTITY_PAGE = "/entity/update";
 
 export const HOOKS_PAGE = "/hooks";
 
-export const ADMIN_DASHBOARD_PAGE = "/admin";
```

Comment:
As you mentioned, lets add a redirect for this url. Sending to `/admin/courses` I think is best, as I believe they would use it more frequently.